TipBot/CommandModules/PublicCommands.cs

```diff
@@ -36,7 +37,38 @@ public class PublicCommands : ModuleBase<SocketCommandContext>
         /// <summary>Protects access to <see cref="CommandsManager"/>.</summary>
         private readonly object lockObject = new object();
 
-        [CommandWithHelp("tip", "Transfers specified amount of money to mentioned user.", "tip <user> <amount> <message>*")]
+		[CommandWithHelp("sc-state", "Inspects smartcontract state at a specific contract and variable", "sc-state string contractAddress, string variableName, string variableType")]
```

Comment:
I'm not sure why do we need smart contracts calling in a tipbot? We have StratisApps for that. 
SC calling doesn't belong here.

Reply:
I don't know what "StratisApps" is, but i definitely need SC calling. They won't look like this in their finished version, they'll be more specific to specific smart contracts, but this is useful for me for testing smart contracts and showing my work on smart contracts in progress to the SDF community.

Reply:
>I don't know what "StratisApps" is

Blazor views that provide UI for smart contracts 

>but this is useful for me for testing smart contracts and showing my work on smart contracts in progress to the SDF community.

In that case this functionality should probably be in a separated repo. This functionality is outside of the scope of the tipbot.